services/Skyra.Cdn/Startup.cs

```diff
@@ -0,0 +1,44 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using Skyra.Database;
+
+namespace Skyra.Cdn
+{
+	public class Startup
+	{
+		public Startup(IConfiguration configuration)
+		{
+			Configuration = configuration;
+		}
+
+		public IConfiguration Configuration { get; }
+
+		// This method gets called by the runtime. Use this method to add services to the container.
+		public void ConfigureServices(IServiceCollection services)
+		{
+			services.AddControllers();
+			services.AddScoped<SkyraDbContext>();
+			services.AddScoped<IDatabase, SkyraDatabase>();
+			services.AddSwaggerGen(c => { c.SwaggerDoc("v1", new OpenApiInfo {Title = "Skyra.Cdn", Version = "v1"}); });
```

Comment:
Are the two "v1"'s here the ""same"" thing? If so, constant pls

Reply:
That's... from the ASP.NET template

Reply:
Please see if it can

Reply:
I just nuked it